pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -46,6 +46,10 @@ const (
 	ConnectionAck = DIDExchangeSpec + "ack"
 	// DIDExchangeServiceType is the service type to be used in DID document
 	DIDExchangeServiceType = "did-communication"
+	// ConnectionID connection id
```

Comment:
suggest describing the field. Similar for L51

Reply:
do these fields need to be exported?

Reply:
@llorllale https://github.com/hyperledger/aries-framework-go/pull/422/files#diff-427522b353a28b1ec350c29c79d387b0R27

Reply:
@troyronda done